PUBLISHING.md

```diff
@@ -37,15 +37,16 @@ Before proceeding we need to generate a new personal token with access to the re
 In the following examples, `cr` is the alias name of the command line tool `chart-releaser`.
```

Comment:
- Please link to chart-releaser web site so that people can install it if they don't have it
- It is better to spell commands in full in docs, i.e. use `chart-releaser` instead of `cr`... it is not that long and people may have different personal alias for `cr`.
- What about moving this to `docs.reana.io` as I commented elsewhere?

Reply:
1. There is a link in the beginning to chart-releaser on GitHub
2. `cr` is the real name of the command-line tool, I will make it clear
3. I am moving it to docs.reana.io now

Reply:
WRT (2), Yes, I just saw separately... Whops! 

Suggested text:

```
In the following examples, `cr` is the [Chart Releaser](...) command-line utility 
allowing to create GitHub releases from Helm charts.
```

Something like that.